src/parser/ad_parser.js

```diff
@@ -90,6 +91,16 @@ function parseAdElement(adTypeElement, emit) {
               apiFramework: creativeElement.getAttribute('apiFramework') || null
             };
 
+            const universalAdIdElement = parserUtils.childByName(
```

Comment:
`Creatives` parsing is becoming rather large. If you could, it would be great to move it to a new file in a function.

Reply:
Moved to a different file, along with the parsing of Extensions as well.